eng/pipelines/stages/build-abstractions-package-ci-stage.yml

```diff
@@ -0,0 +1,351 @@
+#   'packageVersion' and 'buildNumber' parameters, respectively.
+#
+# The following NuGet packages are published:
+#
+#   Microsoft.Data.SqlClient.Extensions.Abstractions.<version>.nupkg
+#   Microsoft.Data.SqlClient.Extensions.Abstractions.<version>.snupkg (symbols)
+#
+# The packages are published to pipeline artifacts with the name specified by
+# the 'artifactName' parameter.
+#
+# This template defines a stage named 'build_abstractions_package_stage' that
+# can be depended on by downstream stages.
+
+parameters:
+  # The type of build to produce (Release or Debug)
+  - name: buildConfiguration
+    displayName: Build Configuration
+    type: string
+    default: Release
+    values:
+    - Release
+    - Debug
+
+  # The version (Major.Minor.Patch) to apply to the package.
+  - name: packageVersion
+    displayName: Package Version Override
+    type: string
+    default: ''
+
+  # The build number of the pipeline.
+  - name: buildNumber
+    displayName: Build Number
+    type: string
+    default: $(Build.BuildNumber)
+
+  # The name of the pipeline artifact to publish.
+  - name: artifactName
+    displayName: Pipeline Artifact Name
+    type: string
+    default: Abstractions.Artifact
+
+  # The list of .NET runtimes to test against.
+  - name: netTestRuntimes
+    displayName: .NET Test Runtimes
+    type: object
+    default: [net8.0, net9.0]
+
+  # The list of .NET Framework runtimes to test against.
+  - name: netFrameworkTestRuntimes
+    displayName: .NET Framework Test Runtimes
+    type: object
+    default: [net462, net47, net471, net472, net48, net481]
+
+  # The verbosity level for the dotnet CLI commands.
+  - name: verbosity
+    displayName: Dotnet CLI verbosity
+    type: string
+    default: normal
+    values:
+    - quiet
+    - minimal
+    - normal
+    - detailed
+    - diagnostic
+
+stages:
+  - stage: build_abstractions_package_stage
+    displayName: Build Abstractions Package
+
+    variables:
+      # The directory where dotnet artifacts will be staged.  Not to be
+      # confused with pipeline artifact.
+      - name: dotnetArtifactsDir
+        value: $(Build.StagingDirectory)/dotnetArtifacts
+
+      # The directory where the NuGet packages will be staged before being
+      # published as pipeline artifacts.
+      - name: dotnetPackagesDir
+        value: $(Build.StagingDirectory)/dotnetPackages
+
+      # The Abstractions solution file to use for all dotnet CLI commands.
+      - name: project
+        value: src/Microsoft.Data.SqlClient.Extensions/Abstractions/Abstractions.slnx
+
+      # dotnet CLI arguments common to all commands.
+      - name: commonArguments
+        value: >-
+          --verbosity ${{parameters.verbosity}}
+          --artifacts-path $(dotnetArtifactsDir)
+
+      # dotnet CLI arguments for build/test/pack commands
+      - name: buildArguments
+        value: >-
+          $(commonArguments)
+          --configuration ${{parameters.buildConfiguration}}
```

Comment:
I know it works to omit the spaces, but the docs for ado pipeline yaml always includes the spaces
ie `${{ parameters.buildConfiguration }}`